wodles/aws/aws_tools.py

```diff
@@ -385,6 +385,8 @@ def get_script_arguments():
                         help='If fail to parse a file, error out instead of skipping the file', default=False)
     parser.add_argument('-o', '--reparse', action='store_true', dest='reparse',
                         help='Parse the log file, even if its been parsed before', default=False)
+    parser.add_argument('-wa', '--waf_acls', dest='waf_acls', help='Comma delimited list of WAF ACLs to fetch',
```

Comment:
I would personally prefer `-w` (or `-waf` if it's possible)

Reply:
I agree that `-w` would be simpler. `-waf` might generate confusion since `waf` is the value used for the `--type` parameter.